meta-flutter/classes/embedded-linux.bbclass

```diff
@@ -0,0 +1,26 @@
+SUMMARY = "Common parts of the recipe for flutter-embedded-linux"
```

Comment:
Common settings of the flutter-embedded-linux recipes

Reply:
I fixed it.